chore: introduce envinfo into environment section (#10553)
babel/babel: .github/ISSUE_TEMPLATE/Bug_report.md

**Expected behavior/code**
A clear and concise description of what you expected to happen (or code).

> **Babel Configuration (.babelrc, babel.config.js, cli command)**

Reviewer: ```suggestion
**Babel Configuration (babel.config.js, .babelrc, package.json#babel, cli command)**
```

```

**Environment**

> <!--- Tryme: you can run `npx envinfo --preset babel` and paste the result below ``` -->

Reviewer 2: I think we can simplify this a bit, with something like:

```suggestion
<!--- Tip: Instead of filling out the questions below, you can run `npx envinfo --preset babel` and paste the result below ``` -->
```

(And remove the subsequent two lines)

```

```
<!--- you can delete the questions below if you have pasted `envinfo` results -->

> <!--- :-) It is also okay if instead you prefer to fill in the information below -->

Reviewer: ```suggestion
<!--- 🙂 It is also okay if instead you prefer to fill in the information below -->
```
